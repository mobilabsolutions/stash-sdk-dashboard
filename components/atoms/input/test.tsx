import { testRender } from '../../../test_utils'

import Input from './input'
import VisibilityIcon from './visibility_icon'

it('Input Should Render', () => testRender(Input))
it('VisibilityIcon Should Render', () => testRender(VisibilityIcon))
