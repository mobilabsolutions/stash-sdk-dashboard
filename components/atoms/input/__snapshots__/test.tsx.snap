// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Should Render 1`] = `
<input
  autoFocus={false}
  className="input__HtmlInput-rcagn3-0 iMczqN"
  onBlur={null}
  onFocus={null}
  placeholder=""
  type="text"
/>
`;

exports[`VisibilityIcon Should Render 1`] = `
<div
  className="visibility_icon__Wrapper-sc-1dur5xf-0 gvfdVz"
>
  <svg
    className="enable"
    height={16}
    onClick={[Function]}
    viewBox="0 0 22 16"
    width={24}
  >
    <path
      d="M11 .5C6 .5 1.73 3.61 0 8c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5C20.27 3.61 16 .5 11 .5zM11 13c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5zm0-8C9.34 5 8 6.34 8 8s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"
      fill="#D6D9DB"
      fillRule="evenodd"
    />
  </svg>
</div>
`;
